lib/http.js

```diff
@@ -127,6 +127,7 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     // TODO: also accept bearer token to identify profile agent in the future
     optionallyAuthenticated,
     // TODO: validate body {profileAgent?}
+    // profileAgent should be a string here
```

Comment:
not sure what this is referring to.  I don't see any variables called `profileAgent`.